src/workerd/io/io-context.c++

```diff
@@ -918,20 +917,22 @@ kj::Own<CacheClient> IoContext::getCacheClient() {
   return getIoChannelFactory().getCache();
 }
 
-jsg::AsyncContextFrame::StorageScope IoContext::makeAsyncTraceScope(jsg::Lock& js) {
-  auto context = js.v8Isolate->GetCurrentContext();
+jsg::AsyncContextFrame::StorageScope IoContext::makeAsyncTraceScope(Worker::Lock& lock) {
+  jsg::Lock& jsgLock = lock;
```

Comment:
nit: by convention we typically use `jsg::Lock& js` as the name... absolutely non-blocking tho.